### PR TITLE
[AVFAudio] Update bindings to Xcode 12.5 Beta 1 to Beta 3

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/avfoundation.cs [hunk 1]
```diff
@@ -2401,6 +2409,14 @@ interface AVAudioSessionDataSourceDescription {
 		
 	}
 
+	[Watch (7,4), NoTV, NoMac, iOS (14,5)]
+	[Native]
+	public enum AVAudioSessionInterruptionReason : ulong {
+		Default = 0,
+		AppWasSuspended = 1,
+		BuiltInMicMuted = 2,
+	}
```
Comment (spouliot) [inline src/avfoundation.cs:2418]: pretty sure I've seen this in @rachelkang PR
AVFAudio is part of AVFoundation

**Review by spouliot — COMMENTED** [review]
LGTM but

File: src/avfoundation.cs (hunk 1 above)
Comment (dalexsoto) [reply src/avfoundation.cs:2418]: Oops 

368C609F-49FA-4B86-94E3-D5AB97CA2449

File: tests/xtro-sharpie/tvOS-AVFoundation.ignore [hunk 2]
```diff
@@ -12,3 +12,6 @@
 # Initial result from new rule missing-release-attribute-on-return-value
 !missing-release-attribute-on-return-value! CoreMedia.CMSampleBuffer AVFoundation.AVAssetReaderOutput::CopyNextSampleBuffer()'s selector's ('copyNextSampleBuffer') Objective-C method family ('copy') indicates that the native method returns a retained object, and as such a '[return: Release]' attribute is required.
 !missing-release-attribute-on-return-value! CoreVideo.CVPixelBuffer AVFoundation.AVVideoCompositionRenderContext::CreatePixelBuffer()'s selector's ('newPixelBuffer') Objective-C method family ('new') indicates that the native method returns a retained object, and as such a '[return: Release]' attribute is required.
+
+# not applicable to tvOS
+!missing-enum! AVAudioSessionInterruptionReason not bound
```
Comment (mandel-macaque) [inline tests/xtro-sharpie/tvOS-AVFoundation.ignore:17]: Add the new line, even when I don't do it.

**Review by mandel-macaque — APPROVED** [review]

**Review by tj-devel709 — APPROVED** [review]

**Comment by dalexsoto** [comment]
Covered by https://github.com/xamarin/xamarin-macios/pull/10886

**Pull request closed without merging**